### PR TITLE
deps: update react-imgix

### PR DESCRIPTION
**Pull request by luqven**
TSIA

### CODE REVIEW

**Comment by luqven** [comment]
Current dependencies on/for this PR:
* next
  * **PR #140** <a href="https://app.graphite.dev/github/pr/imgix/contentful/140" target="_blank">Graphite</a>  👈
    * **PR #139** <a href="https://app.graphite.dev/github/pr/imgix/contentful/139" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/imgix/contentful/140?utm_source=stack-comment).

**Review by sherwinski — APPROVED** [review]

**Comment by luqven** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/imgix/contentful/140?utm_source=stack-comment).


Job Reference: yGgqIySrcaAmVh8uxOCg

**Pull request merged**
